templates/components/category/shop-by-price.html

```diff
@@ -1,4 +1,4 @@
-{{#if shop_by_price}}
+{{#if theme_settings.shop_by_price}}
```

Comment:
Derp. Completely overlooked that. I'll get it added as well as update the changelog. Thanks, Mario!